src/common/create_task_readme/render_all.sh

```diff
@@ -5,4 +5,4 @@ TASK_IDS=`ls src/tasks`
 for task_id in $TASK_IDS; do
   echo ">> Processing $task_id"
   viash run src/common/create_task_readme/config.vsh.yaml -- --task $task_id
-done
```

Comment:
Please undo this change

```suggestion
done

```

Reply:
noted